README.md

```diff
@@ -93,6 +93,23 @@ This gem also includes a dependency on `rubocop-rspec` that will be updated to
 the latest compatible version each time that the MAJOR.MINOR version of `rubocop` 
 is updated.
 
+## Change Process
+
+This configuration is meant to represent the general opinion of Salsify's Ruby community around best practices for
+writing readable Ruby code. As a result, changes to this configuration should go through a discussion phase in the
```

Comment:
Should we document our philosophy around which cops to enable e.g. generally enable cops with auto-correct but be skeptical of cops (especially stylistic ones) without auto-correct?

Reply:
I added some discussion of our philosophy—let me know if you think of other points worth mentioning.